src/itrim.cpp

```diff
@@ -110,6 +110,11 @@ usage() {
         "these options are used, the replicas will be trimmed until there are",
         "numCopies left. The oldest replicas will be trimmed first.",
         " ",
+        "It is recommended that this iCommand be used as a precision tool for managing",
+        "individual replicas. First, determine the replica number of the replica to",
+        "delete, then target that replica for deletion. Here is the usage:",
+        "    itrim -n 2 /tempZone/home/alice/foo",
```

Comment:
Let's add an empty line between these lines so the command stands out more.

Reply:
Done